Tools/autotest/common.py

```diff
@@ -1421,117 +1421,108 @@ def wait_seconds(self, seconds_to_wait):
         while tstart + seconds_to_wait > tnow:
             tnow = self.get_sim_time()
 
-    def wait_altitude(self, alt_min, alt_max, timeout=30, relative=False):
```

Comment:
This separate min/max thing was actually being used and being useful.  I don't think we can lose it.

Reply:
ok I will change that one